### PR TITLE
[SPARK-18575][WEB] Keep same style: adjust the position of driver log links

### PR DESCRIPTION
**Pull request by uncleGen**
## What changes were proposed in this pull request?

NOT BUG, just adjust the position of driver log link to keep the same style with other executors log link.

image

## How was this patch tested?
 no

### CODE REVIEW

File: yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClusterSchedulerBackend.scala [hunk 1]
```diff
@@ -55,8 +55,8 @@ private[spark] class YarnClusterSchedulerBackend(
       val baseUrl = s"$httpScheme$httpAddress/node/containerlogs/$containerId/$user"
       logDebug(s"Base URL for logs: $baseUrl")
       driverLogs = Some(Map(
-        "stderr" -> s"$baseUrl/stderr?start=-4096",
-        "stdout" -> s"$baseUrl/stdout?start=-4096"))
+        "stdout" -> s"$baseUrl/stdout?start=-4096",
```
Comment (srowen) [inline yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClusterSchedulerBackend.scala]: Explain please what this is trying to be consistent with?

Comment (uncleGen) [inline yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClusterSchedulerBackend.scala]: executor log link order:  **stdout** then **stderr**
but, the driver log link order: **stderr** then **stdout**

After PR:

image

Frankly, it is just a picky issue.

**Comment by uncleGen** [comment]
retest this please

**Comment by ajbozarth** [comment]
Not sure this is worth a whole commit, but it is probably good to fix. LGTM.

**Comment by uncleGen** [comment]
@srowen waiting for your feedback :D

**Review by srowen — APPROVED** [review]
OK I see it now, sounds fine.

**Comment by srowen** [comment]
Merged to master/2.1

**Pull request closed without merging**